tensorflow/c/c_api.cc

```diff
@@ -154,6 +155,20 @@ void TF_DeleteBuffer(TF_Buffer* buffer) {
 
 TF_Buffer TF_GetBuffer(TF_Buffer* buffer) { return *buffer; }
 
+void TF_TensorFromProto(const TF_Buffer* from, TF_Tensor* to,
+                        TF_Status* status) {
+  TF_SetStatus(status, TF_OK, "");
+  tensorflow::TensorProto from_tensor_proto;
+  Status cc_status;
```

Comment:
We could avoid creating `cc_status` here I think:

```
status->status = BufferToMessage(from, &from_tensor_proto);
if (!status->status.ok()) {
  return;
} 
status->status = tensorflow::down_cast<tensorflow::TensorInterface*>(to->tensor)->FromProto(from_tensor_proto);
```

Reply:
Yes, that makes sense. Done.